src/CSSTransition.js

```diff
@@ -150,44 +141,62 @@ class CSSTransition extends React.Component {
     const isStringClassNames = typeof classNames === 'string';
     const prefix = isStringClassNames && classNames ? classNames + '-' : '';
 
-    let className = isStringClassNames ?
+    let baseClassName = isStringClassNames ?
       prefix + type : classNames[type]
 
     let activeClassName = isStringClassNames ?
-      className + '-active' : classNames[type + 'Active'];
+      baseClassName + '-active' : classNames[type + 'Active'];
 
     let doneClassName = isStringClassNames ?
-      className + '-done' : classNames[type + 'Done'];
+      baseClassName + '-done' : classNames[type + 'Done'];
 
     return {
-      className,
+      baseClassName,
       activeClassName,
       doneClassName
     };
   }
 
-  removeClasses(node, type) {
-    const { className, activeClassName, doneClassName } = this.getClassNames(type)
-    className && removeClass(node, className);
-    activeClassName && removeClass(node, activeClassName);
-    doneClassName && removeClass(node, doneClassName);
-  }
+  addClass(node, type, phase = 'base') {
```

Comment:
```suggestion
  addClass(node, type) {
```

given how you're using this, i think it's a little cleaner to always pass in the `phase` explicitly. the default doesn't really feel "special" to me; it's just one of three possible phases, so i think it will be a little easier to read the code if we're always specifying the phase explicitly